diff --git a/node_modules/react-native-paper/src/components/BottomNavigation.js b/node_modules/react-native-paper/src/components/BottomNavigation.js
index 28dff89..20c4e50 100644
--- a/node_modules/react-native-paper/src/components/BottomNavigation.js
+++ b/node_modules/react-native-paper/src/components/BottomNavigation.js
@@ -673,6 +673,10 @@ class BottomNavigation<T: *> extends React.Component<Props<T>, State> {
                 }
               : null,
             barStyle,
+            {
+              borderColor: theme.colors.borderColor,
+              borderTopWidth: StyleSheet.hairlineWidth,
+            }
           ]}
           pointerEvents={
             keyboardHidesNavigationBar && this.state.keyboard ? 'none' : 'auto'
diff --git a/node_modules/react-native-paper/src/components/Card/Card.js b/node_modules/react-native-paper/src/components/Card/Card.js
index 6789ad9..a1c156c 100644
--- a/node_modules/react-native-paper/src/components/Card/Card.js
+++ b/node_modules/react-native-paper/src/components/Card/Card.js
@@ -4,6 +4,7 @@ import * as React from 'react';
 import {
   Animated,
   View,
+  Platform,
   TouchableWithoutFeedback,
   StyleSheet,
 } from 'react-native';
@@ -93,7 +94,7 @@ class Card extends React.Component<Props, State> {
   static Title = CardTitle;
 
   static defaultProps = {
-    elevation: 1,
+    elevation: Platform.OS === 'android' && Platform.Version >= 28 ? 0 : 1,
   };
 
   state = {
